app/twitter_service.py

```diff
@@ -13,8 +13,8 @@
 def twitter_api():
     auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
     auth.set_access_token(ACCESS_KEY, ACCESS_SECRET)
-    api = tweepy.API(auth)
-    return api # consider returning the auth as well
+    api = tweepy.API(auth, wait_on_rate_limit=True, wait_on_rate_limit_notify=True)
```

Comment:
this change is theoretically helpful, although not yet able to observe its effect.